rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -590,7 +590,7 @@ bool gs_frame::event(QEvent* ev)
 		}
 		close();
 	}
-	else if (ev->type() == QEvent::MouseMove && !m_show_mouse)
+	else if (ev->type() == QEvent::MouseMove && (!m_show_mouse || m_mousehide_timer.isActive()))
```

Comment:
Does it need these conditions?
If handle_cursor(visibility(), false, true) could be called only with the mouse move condition it might solve this issue, and reduce complexity, OR.. even better, if perhaps handle_cursor was split into two functions restart_mouse_idle_timer(), and handle_cursor() [at the moment I think we've coupled two functions which shouldn't be so coupled into one..]

Although I also think that my split function idea isn't not worth the effort if this fix gets it across the line :)

Reply:
haha... and the boat sailed before I posted my comment :P  lol
Ohh well, if it's fixed, it's fixed :)